app/quest-boost/[boostId]/page.tsx

```diff
@@ -129,55 +136,69 @@ export default function Page({ params }: BoostQuestPageProps) {
-                  expired={quest.expired}
-                />
-              );
-            })}
-          </div>
-          <div className={styles.claim_button_container}>
-            <div className={styles.claim_button_text_content}>
-              <p>Reward:</p>
-              <div className={questStyles.issuer}>
-                <p>
-                  {boost?.amount} {getTokenName(boost?.token ?? "")}
-                </p>
-                <TokenSymbol tokenAddress={boost?.token ?? ""} />
+          {boost && (
+            <>
+              <div className="flex flex-col">
+                <h1 className={styles.title}>{boost?.name}</h1>
+                {boost?.expiry && boost.expiry > Date.now() ? (
+                  <Timer fixed={false} expiry={Number(boost?.expiry)} />
+                ) : null}
+              </div>
+
+              <div className={styles.card_container}>
+                {quests?.map((quest, index) => {
+                  if (quest?.hidden || quest?.disabled) return null;
+                  return (
+                    <Quest
+                      key={index}
+                      title={quest.title_card}
+                      onClick={() => router.push(`/quest/${quest.id}`)}
+                      imgSrc={quest.img_card}
+                      issuer={{
+                        name: quest.issuer,
+                        logoFavicon: quest.logo,
+                      }}
+                      reward={quest.rewards_title}
+                      id={quest.id}
+                      expired={quest.expired}
+                    />
+                  );
+                })}
               </div>
-              <p>among</p>
-              <p className={styles.claim_button_text_highlight}>
-                {participants} players
-              </p>
-            </div>
-            {address ? (
-              <div>
-                <Button disabled={buttonDisabled} onClick={handleButtonClick}>
-                  {getButtonText()}
-                </Button>
+              <div className={styles.claim_button_container}>
+                <div className={styles.claim_button_text_content}>
+                  <p>Reward:</p>
+                  <div className={questStyles.issuer}>
+                    <p>
+                      {boost?.amount} {getTokenName(boost?.token ?? "")}
+                    </p>
+                    <TokenSymbol tokenAddress={boost?.token ?? ""} />
+                  </div>
+                  <p>among</p>
+                  <p className={styles.claim_button_text_highlight}>
+                    {participants} players
+                  </p>
+                </div>
+                {address ? (
+                  <div>
+                    <Button
+                      disabled={buttonDisabled}
+                      onClick={handleButtonClick}
+                    >
+                      {getButtonText()}
+                    </Button>
+                  </div>
+                ) : null}
```

Comment:
also i think a better solution would be to add error screen component here instead of placing a null. Do different if else covering same conditions won't make much sense

Reply:
This particular condition is dynamically rendering a button, so if the condition is not met nothing is rendered.

Reply:
actually sorry i commented on this line. I meant line 139. 
instead of doing `boost &&  X` and `!boost && Y` you can do `boost ? return X if true : return Y if true`